### ISSUE TITLE
Rotate the preview video

### ISSUE THREAD

**Issue by koppa** [mifi/lossless-cut#91]
It would be awesome to rotate the preview view to the rotation settings

A quick and dirty fix is:
```
diff --git a/src/renderer.jsx b/src/renderer.jsx
index 6449654..36b0353 100644
--- a/src/renderer.jsx
+++ b/src/renderer.jsx
@@ -116,6 +116,8 @@ class App extends React.Component {
       video.currentTime = 0;
       video.playbackRate = 1;
       this.setState(localState);
+
+      player.children[0].style.transform = "";
     };
 
     const load = (filePath, html5FriendlyPath) => {
@@ -257,6 +259,8 @@ class App extends React.Component {
   increaseRotation() {
     const rotation = (this.state.rotation + 90) % 450;
     this.setState({ rotation });
+
+    player.children[0].style.transform = "rotate("+rotation+"deg)";
   }
```

This change works perfectly on my machine.

I would also prefer to remember the settings when opening a new video, instead of resetting them to default.

**Comment by mifi** [comment]
I tried this myself, but I think the problem is that this rotation is relative to the rotation already applied by the html5 player (from the file). So if the file is rotated 180 deg in the file, and you rotate the video 180 deg, and apply this transform, then it would looks like the video is 180+180=360 deg, but in reality it is 180.

currently these settings are remembered when opening a new file:
```
      stripAudio,
      includeAllStreams,
      captureFormat,
      customOutDir,
      keyframeCut,
```
You mean you would like also rotation to be remembered? I think usually rotation will vary from file to file, so I don't see why that's helpful?

**Comment by koppa** [comment]
I haven't tested it with already rotated video files.

My use case is, that I have maybe 10 GB of video footage split into many smaller files and need to cut the interesting clips out. In many of these videos the footage was filmed upside down. So for each file I have to press the rotate video button multiple times. Keeping the rotation value across multiple loads would simplify my workflow.

**Comment by mifi** [comment]
I agree! I had the same problem with my own videos hence why i added this function, but i could not find a way to present the output rotation.

**Comment by mifi** [comment]
We could implement a dialog that presents a single frame of the video rendered to JPG and shown in an <img /> after applying rotation.

**Comment by mifi** [comment]
I have now fixed this in 1.5.0
